crates/extensions/tedge_downloader_ext/src/tests.rs

```diff
@@ -12,7 +12,7 @@ const TEST_TIMEOUT: Duration = Duration::from_secs(5);
 #[tokio::test]
 async fn download_without_auth() {
     let ttd = TempTedgeDir::new();
-    let mut server = mockito::Server::new();
+    let mut server = mockito::Server::new_async().await;
```

Comment:
The same comment as `extensions/c8y_auth_proxy/src/server.rs`.
Don't we need to change from crete() to create_async() in this file?

Reply:
https://github.com/thin-edge/thin-edge.io/pull/3216#discussion_r1822733495